playbooks/roles/openvpn/templates/openvpn-iptables.service.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Set the firewall rules required for openvpn
```

Comment:
I think `Description=Firewall rules required for OpenVPN` would be a little bit clearer. Right now this shows up a little wonky in `journalctl` output:
>`systemd[1]: Started Set the firewall rules required for openvpn`